src/recommendations/RecommendationsPage.jsx

```diff
@@ -1,87 +1,29 @@
-    objectId: 'course-4c70ad9b-9602-49af-bf00-83fa4bf47708',
-    owners: [
-      {
-        key: 'MITx',
-        logoImageUrl: 'https://prod-discovery.edx-cdn.org/organization/logos/2a73d2ce-c34a-4e08-8223-83bca9d2f01d-2cc8854c6fee.png',
-        name: 'Massachusetts Institute of Technology',
-      },
-    ],
-    title: 'Machine Learning with Python: from Linear Models to Deep Learning',
-  },
-  {
-    activeRunKey: 'course-v1:MITx+6.86x+1T2023',
-    cardImageUrl: 'https://prod-discovery.edx-cdn.org/media/course/image/4c70ad9b-9602-49af-bf00-83fa4bf47708-dc4566d15250.jpg',
-    marketingUrl:
-    'https://www.edx.org/course/machine-learning-with-python-from-linear-models-to',
-    objectId: 'course-4c70ad9b-9602-49af-bf00-83fa4bf47708',
-    owners: [
-      {
-        key: 'MITx',
-        logoImageUrl: 'https://prod-discovery.edx-cdn.org/organization/logos/2a73d2ce-c34a-4e08-8223-83bca9d2f01d-2cc8854c6fee.png',
-        name: 'Massachusetts Institute of Technology',
-      },
-      {
-        key: 'MITx',
-        logoImageUrl: 'https://prod-discovery.edx-cdn.org/organization/logos/2a73d2ce-c34a-4e08-8223-83bca9d2f01d-2cc8854c6fee.png',
-        name: 'Massachusetts Institute of Technology',
-      },
-    ],
-    title: 'Machine Learning with Python: from Linear Models to Deep Learning',
-  },
-  {
-    activeRunKey: 'course-v1:MITx+6.86x+1T2023',
-    cardImageUrl: 'https://prod-discovery.edx-cdn.org/media/course/image/4c70ad9b-9602-49af-bf00-83fa4bf47708-dc4566d15250.jpg',
-    marketingUrl:
-    'https://www.edx.org/course/machine-learning-with-python-from-linear-models-to',
-    objectId: 'course-4c70ad9b-9602-49af-bf00-83fa4bf47708',
-    owners: [
-      {
-        key: 'MITx',
-        logoImageUrl: 'https://prod-discovery.edx-cdn.org/organization/logos/2a73d2ce-c34a-4e08-8223-83bca9d2f01d-2cc8854c6fee.png',
-        name: 'Massachusetts Institute of Technology',
-      },
-    ],
-    title: 'Machine Learning with Python: from Linear Models to Deep Learning',
-  },
-  {
-    activeRunKey: 'course-v1:MITx+6.86x+1T2023',
-    cardImageUrl: 'https://prod-discovery.edx-cdn.org/media/course/image/4c70ad9b-9602-49af-bf00-83fa4bf47708-dc4566d15250.jpg',
-    marketingUrl:
-    'https://www.edx.org/course/machine-learning-with-python-from-linear-models-to',
-    objectId: 'course-4c70ad9b-9602-49af-bf00-83fa4bf47708',
-    owners: [
-      {
-        key: 'MITx',
-        logoImageUrl: 'https://prod-discovery.edx-cdn.org/organization/logos/2a73d2ce-c34a-4e08-8223-83bca9d2f01d-2cc8854c6fee.png',
-        name: 'Massachusetts Institute of Technology',
-      },
-    ],
-    title: 'Machine Learning with Python: from Linear Models to Deep Learning',
-  },
-];
-
 const RecommendationsPage = (props) => {
-  const { intl, location } = props;
+  const { intl, personalizedRecommendations, location } = props;
+  const { recommendations, isLoading, showError } = personalizedRecommendations;
```

Comment:
we can also combine there two destructing. Could you please use nested destructing?